pkg/scheduler/preemption/preemption.go

```diff
@@ -162,8 +162,21 @@ func canBorrowWithinCohort(cq *cache.ClusterQueue, wl *kueue.Workload) (bool, *i
 	return true, &threshold
 }
 
+const (
+	// ClusterQueueOrigin indicates that preemption originated from cluster queue
+	ClusterQueueOrigin = "ClusterQueue"
+	// CohortOrigin indicates that preemption originated from cohort
+	CohortOrigin = "cohort"
+
+	InClusterQueueReason = "InClusterQueue"
+
+	PriorityReclamationReason = "PriorityReclamation"
+	InCohortReclamationReason = "InCohortReclamation"
+	InCohortFairSharingReason = "InCohortFairSharing"
```

Comment:
```suggestion
	InCohortReclamationEviction = "InCohortReclamation"
	InCohortFairSharingEviction = "InCohortFairSharing"
```

 InClusterQueueReason is sufficient